1_files_windows/README.md

```diff
@@ -54,22 +54,70 @@ C-ws                horizontal split
 :vs                 vertical split
 C-wv                vertical split
 C-wo                close all other windows
+C-ww                toggle between windows
+C-wl                move to the window on the right
+C-wh                move to the window on the left
 ```
 Splits can be moved and resized as well.  Without any other arguments,
 the split contains the same active buffer, but you can also specify a filename
 to `:sp` or `:vs` to open a new file in a split.  Opening the same buffer is
 handy for large files or when you need to look at an implementation while using
 it.
 
+> __Exercise 1__:
+>
+>    Make a vertical split of this buffer using :vs then try deleting word 
+>
+>    It should update in both windows! Then C-wo to close the vertical split
+
+> __Exercise 2__:
+>
+>    Make a horizontal split of file2.txt using :sp file2.txt
+
 A set of windows are contained in a tab.  You can work with tabs with ex commands:
 ```
 :tabe <FILE>         open a new tab with file or an empty buffer
 :tabe %              open a new tab with current file
 :tabc                close this tab
 :tabo                close all other tabs
-gt,gT               cycle through tabs
+gt,gT                cycle through tabs
 ```
+
+> __Exercise 3__:
+>
+>  Make a new tab of file1.txt using `:tabe file1.txt`
+>
+>  Then follow the directions there, go ahead and do that now.
+>
+>  After you end up back here use gT to go back one more time!
+>
+>  Phew, welcome back
+
 The buffer, window, tab model of vim is slightly different than an IDE.  A
 tab is a collection of windows.  A window is a view on a buffer. A buffer is
 an in-memory view of a file.  Once you have a window you can change it to any
 buffer.
+
+## Multiple files and reading from stdin
+
+You can open multiple files at once with vim from the command line using
+wildcards such as `vi *.txt`. By default, this does not open each file in a
+separate tab, and instead you can move from one file to the next with :n or :N.
+You can also use `:tab all` to open each buffer in a separate tab.
```

Comment:
Would probably be a good place to mention the `-o`, `-O`, and `-p` options here for dealing with multiple files.